sktime/transformations/series/peak.py

```diff
@@ -0,0 +1,569 @@
+        first end week, the second one determines the second end week and so on.
+        [peak_week_end1, peak_week_end2, peak_week_end3, ...]
+    peak_month_start : list, default= None
+        start interval of peak month(s) in a list where the first argument determines
+        the first start month, the second one determines the second start month
+        and so on. [peak_month_start1, peak_month_start2, peak_month_start3, ...]
+    peak_month_end : list, default= None
+        end interval of peak month(s) in a list where the first argument determines the
+        first end month, the second one determines the second end month and so on.
+        [peak_month_end1, peak_month_end2, peak_month_end3, ...]
+    peak_quarter_start : list, default= None
+        start interval of peak quarter(s) in a list where the first argument determines
+        first start quarter, the second one determines the second start quarter
+        and so on. [peak_quarter_start1, peak_quarter_start2, peak_quarter_start3, ...]
+    peak_quarter_end : list, default= None
+        end interval of peak quarter(s) in a list where the first argument determines
+        first end quarter, the second one determines the second end quarter and so on.
+        [peak_quarter_end1, peak_quarter_end2, peak_quarter_end3, ...]
+    peak_year_start : list, default= None
+        start interval of peak year(s) in a list where the first argument determines
+        first start year, the second one determines the second start year and so on.
+        [peak_year_start1, peak_year_start2, peak_year_start3, ...]
+    peak_year_end : list, default= None
+        end interval of peak year(s) in a list where the first argument determines
+        first end year, the second one determines the second end year and so on.
+        [peak_year_end1, peak_year_end2, peak_year_end3, ...]
+    working_hour_start : list, default= None
+        start interval of working hour(s) in a list where the first argument determines
+        first start working hour, the second one determines the second start working
+        hour and so on.
+        e.g., [working_hour_start1, working_hour_start2, working_hour_start3, ...]
+    working_hour_end : list, default= None
+        end interval of working hour(s) in a list where the first argument determines
+        first end working hour, the second one determines the second end working hour
+        and so on. [working_hour_end1, working_hour_end2, working_hour_end3, ...]
+    keep_original_columns :  boolean, optional, default=False
+        If True, keep original columns in main dataframe (X) passed to `.transform()`.
+    keep_original_peaktime_data_columns: boolean, optional, default=False
+        If True, keep original peaktime_data dataframe columns including all separate
+        peak/working columns, e.g., peak_hour_1, peak_hour_2, peak_week_1,
+        peak_week_2, ...
+
+    Returns
```

Comment:
remove the Returns block, this is not correct (it's a class)

Reply:
Thanks! Done!